composer.json

```diff
@@ -64,8 +64,8 @@
         "yiisoft/yii-middleware": "dev-master",
         "yiisoft/yii-queue": "3.0.x-dev",
         "yiisoft/yii-runner-console": "^1.0",
-        "yiisoft/yii-runner-http": "^1.0",
-        "yiisoft/yii-swagger": "^1.0"
+        "yiisoft/yii-runner-http": "dev-master",
+        "yiisoft/yii-swagger": "dev-master"
```

Comment:
I think we'd better release these.

Reply:
There are no changes in yii-runner-http compared to latest release. Let's use stable version.

Reply:
I've tagged yii-swagger as well so let's specify stable version.

Reply:
Did it.